docker-compose.yml

```diff
@@ -5,10 +5,11 @@ services:
     container_name: auv_ws
     image: ghcr.io/ufrj-nautilus/auv_ws:master
     privileged: true
+    pull_policy: always
     environment:
       - DISPLAY=${DISPLAY} 
     volumes:
-      - ./../../../catkin_ws:/root/catkin_ws
+      - ./../catkin_ws:/root/catkin_ws
```

Comment:
Change this to mount the catkin_ws/src in the container.
Probably this:
```
- ./../src/:/root/catkin_ws/src
```

We don't need build and devel outside the container anyway.